tests/utils/command.go

```diff
@@ -43,3 +48,46 @@ func RunCommand(bin string, args ...string) (*cmd.Cmd, error) {
 
 	return curCmd, nil
 }
+
+func RegisterPingTest() {
+	ginkgo.Describe("[Precompiles]", ginkgo.Ordered, func() {
```

Comment:
should the reference to `[Precompiles]` still be here?

Reply:
ah good catch, no need for that.